crates/bootstrap_srv/certs/test_cert.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
How long are these valid for? We could instead use tempfile and [rcgen](https://docs.rs/rcgen/latest/rcgen/fn.generate_simple_self_signed.html).

Reply:
3 years but yes, can use `rcgen` and it should keep working